applications/kommander-flux/2.6.2/helmrelease/helmrelease.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: source.toolkit.fluxcd.io/v1
+kind: OCIRepository
+metadata:
+  name: flux2
+  namespace: ${releaseNamespace}
+spec:
+  interval: 1m
+  url: "${ociRegistryURL:=oci://ghcr.io}/fluxcd-community/charts/flux2"
+  ref:
+    tag: 2.16.1
+---
+apiVersion: helm.toolkit.fluxcd.io/v2
+kind: HelmRelease
+metadata:
+  name: kommander-flux
+  namespace: ${releaseNamespace}
+spec:
+  chartRef:
+    kind: OCIRepository
+    name: flux2
+    namespace: ${releaseNamespace}
+  interval: 15s
+  install:
```

Comment:
maybe we can use 

```
install:
  strategy: RetryOnFailure
  retryInterval: 5m
```
same for upgrades too.

We should NOT use a remediation strategy - our only option is to keep retrying for ever. Remediation might result in uninstalls.